.github/workflows/pr.yml

```diff
@@ -10,12 +10,14 @@ jobs:
 
     steps:
       - name: "Checkout source"
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
+        with:
+          submodules: true
 
       - name: "Build artifacts"
         run: |
-          ARCH=rm make docker
-          ARCH=kobo make docker
+          CROSS_TC=arm-kobo-linux-gnueabihf DOCKERFILE=Dockerfile.kobo FBINK=1 TARGET=kobo make docker
```

Comment:
not sure we need this change or if using target=kobo is enough

Reply:
I was copying what was in the stable-release action. Should they both be changed to just `TARGET=kobo`?
https://github.com/rmkit-dev/rmkit/blob/381ef9d52742ebf2861ec6d31e2aeaa590a67bfd/.github/workflows/stable-release.yml#L29-L30

Reply:
ah, thanks for fixing it here,  then. I think i trust stable release,  since that's been running and uploading artifacts